packages/machine/src/models/app-instance.ts

```diff
@@ -105,6 +105,10 @@ export class AppInstance {
     return ret;
   }
 
+  public toJson(): AppInstanceJson {
+    return this.json;
```

Comment:
I meant to inquire with @snario about the pattern of using `this.json` to store the state of the AppInstance. Can you explain why this is preferred over setting them on as instance properties?

Reply:
We discussed this on a call

Reply:
Keeping it as is?

Reply:
For now yes though we have an improvement plan for a future PR, not urgent.